opal/datatype/opal_datatype_pack.h

```diff
@@ -106,8 +106,8 @@ static inline void pack_predefined_data(opal_convertor_t *CONVERTOR, const dt_el
     *(COUNT) -= cando_count;
 
     if (_elem->blocklen < 9) {
-        if ( !(CONVERTOR->flags & CONVERTOR_CUDA) &&
-             !(CONVERTOR->flags & CONVERTOR_ROCM)
+        if (!(CONVERTOR->flags & CONVERTOR_ACCELERATOR) &&
+            !(CONVERTOR->flags & CONVERTOR_ROCM)
             && OPAL_LIKELY(
```

Comment:
Do we really need the CONVERTOR->flags & CONVERTOR_ROCM or is it just a stop-gap measure for now? (same question applies to a couple of other places in opal/datatype).

Reply:
Stop gap

Reply:
ew; this needs to go.  We shouldn't have a stop-gap like this.